packages/search-ui-app-search-connector/src/__tests__/AppSearchAPIConnector.test.js

```diff
@@ -340,37 +340,41 @@ describe("AppSearchAPIConnector", () => {
       });
     });
 
-    it("will use the additionalOptions parameter to append additional parameters to the search endpoint call", async () => {
+    it("will use the beforeSearchCall parameter to amend option parameters to the search endpoint call", async () => {
```

Comment:
From my understanding of the PR/changes, amending should allow you to remove options from the current obj, correct? Is there any chance we could add/expand these tests to show a value being removed, not just a value being added? That would help my understanding of that functionality personally!

Reply:
Yes, great point.